host-controller/src/main/java/org/jboss/as/host/controller/operations/NativeManagementRemoveHandler.java

```diff
@@ -42,7 +42,7 @@
     public static final NativeManagementRemoveHandler INSTANCE = new NativeManagementRemoveHandler();
 
     public NativeManagementRemoveHandler() {
-        super(NATIVE_MANAGEMENT_CAPABILITY);
+        super(NATIVE_MANAGEMENT_RUNTIME_CAPABILITY);
```

Comment:
Further note to self -- this class is unused as it got dropped in a previous elytron mega-commit. So I'll need to sort that.